### PR TITLE
[OPS-7611] Add the git pre-commit hooks and composer script to install them. 

### PR DESCRIPTION
**Pull request by cafuego**
Also drop in theme asset scripts, we will need them I suspect.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -70,7 +70,22 @@
         "release": "conventional-changelog --commit",
         "release:patch": "conventional-changelog --patch --commit",
         "release:minor": "conventional-changelog --minor --commit",
-        "release:major": "conventional-changelog --major --commit"
+        "release:major": "conventional-changelog --major --commit",
+        "git-hooks": "git config core.hooksPath git-hooks || exit 0",
+        "sass-prepare": "cd html/themes/custom/gms_ocha_common && npm install",
+        "sass-build": "cd html/themes/custom/gms_ocha_common && ./node_modules/.bin/sass sass:build",
+        "sass-cleanup": "cd html/themes/custom/gms_ocha_common && rm -rf ./node_modules",
```
Comment (attiks) [inline composer.json:77]: Theme is located at html/sites/all/themes/ocha_basic/

**Review by attiks — APPROVED** [review]
Aassuming build process is not compiling sass

File: composer.json (hunk 1 above)
Comment (cafuego) [reply composer.json:77]: I hope not, that's the Drupal 7 theme.

Comment (lazysoundsystem) [reply composer.json:77]: There's no subtheme yet in this repo - from gms or from the d7 version.

**Review by lazysoundsystem — APPROVED** [review]
Needs three changes (I think) to pass the travis tests:

Remove the installed_paths config line: https://github.com/UN-OCHA/hrinfo-site-2022/blob/c8180ca9a15093d077930dfaa262a37f46b72388/.travis.yml#L45

add `// @codingStandardsIgnoreStart` at the top of CalFileParser.php https://github.com/UN-OCHA/hrinfo-site-2022/blob/c8180ca9a15093d077930dfaa262a37f46b72388/html/modules/custom/hr_paragraphs/src/Controller/CalFileParser.php#L3

Run phpcbf and fix the remaining unserialize notice on https://github.com/UN-OCHA/hrinfo-site-2022/blob/c8180ca9a15093d077930dfaa262a37f46b72388/html/modules/custom/ocha_docstore_files/src/Plugin/search_api/processor/StoreEntity.php#L214

File: composer.json (hunk 1 above)
Comment (cafuego) [reply composer.json:77]: Yup, I just pasted in the sass script commands so they're there for when we do have a sub-theme. Nothing will try to actually run them at this point.

**Pull request merged**